README.md

```diff
@@ -105,7 +105,8 @@ merged into the RFC repository as a markdown file. At that point the RFC is
 
   - Fork the RFC repo [RFC repository]
   - Copy `0000-template.md` to `text/0000-my-feature.md` (where "my-feature" is
-    descriptive. don't assign an RFC number yet).
+    descriptive. Don't assign an RFC number yet; This is going to be the PR number
+    and we'll rename the file accordingly if the RFC is accepted).
```

Comment:
How about this @nikomatsakis ? It specifies both the "when" and "how" in a concise way.

Reply:
Seems good, thanks. One final nit: can you move the closing parenthesis to "(where "my-feature is descriptive). Don't assign...", so that the sentence doesn't end mid-parenthetical?